src/index.html

```diff
@@ -8,6 +8,32 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <script type="application/ld+json">
+  {
+  "@context": "http://schema.org",
+  "@type": "Organization",
+  "url": "https://dockstore.org",
```

Comment:
Just going to throw this out there, since I wasn't even aware until this PR that search engines looked for this -- should the host be dynamic, e.g., staging.dockstore.org, or in theory, if not likely, somebody tried to stand up their own instance of Dockstore?

Since you're serving up the index.html as is -- it's not a template where you can substitute values at runtime, even if my point is valid (?), it's probably more trouble than it's worth.

Reply:
I think, it depends. 
For staging, I don't actually really want it to rank highly. I might even put in a robots.txt to exclude it if it starts ranking highly. Similarly, for this "overall" information, I'd actually want the search engine to essentially redirect/strongly nudge someone to the main site. 

For someone standing up their own site, it's true. But I think we/they can put more resources into that when the time comes.